src/uct/ib/rc/accel/rc_mlx5.inl

```diff
@@ -919,6 +919,30 @@ static UCS_F_ALWAYS_INLINE void uct_rc_mlx5_common_txqp_bcopy_post(
     uct_rc_txqp_add_send_op(txqp, &desc->super);
 }
 
+static UCS_F_ALWAYS_INLINE uint16_t
+uct_rc_mlx5_common_iface_txwq_update_tx_res(uct_rc_iface_t *rc_iface,
+                                            uct_ib_mlx5_txwq_t *txwq,
+                                            uct_rc_txqp_t *txqp,
+                                            uint16_t hw_ci)
+{
+    uint16_t bb_num = uct_ib_mlx5_txwq_update_bb(txwq, hw_ci) -
+                      uct_rc_txqp_available(txqp);
```

Comment:
seems like there is extra math for dc case now

Reply:
yes, then it means that they can't be moved to common code